spec/lib/algolia_places_client_spec.rb

```diff
@@ -89,6 +89,21 @@
       expect(json[:state]).to eq("Ciudad de México")
       expect(json[:postcodes]).to include("06140")
     end
+
+    xit "works without webmock" do
```

Comment:
I'm leaving this test here to mock with the actual Algolia integration. it's disabled because our API keys have limits and this test can become flaky.